$NetBSD$

Make prefix settable from the environment.
Use the correct python binary.

--- Makefile.orig	2014-12-15 00:07:20.000000000 +0000
+++ Makefile
@@ -1,6 +1,6 @@
 version := $(strip $(shell cat VERSION))
 series := $(strip $(shell cat SERIES))
-prefix := /usr/local
+prefix ?= /usr/pkg
 build_dir := "."
 dist_name = fs-uae-launcher-$(version)
 dist_dir := $(build_dir)/$(dist_name)
@@ -86,7 +86,7 @@ mo: $(catalogs)
 
 
 install-program:
-		python3 setup.py install --root=$(DESTDIR)		 --install-lib=$(prefix)/share/fs-uae-launcher		 --install-scripts=$(prefix)/share/fs-uae-launcher
+	${PYTHONBIN} setup.py install --root=$(DESTDIR)		 --install-lib=$(prefix)/share/fs-uae-launcher		 --install-scripts=$(prefix)/share/fs-uae-launcher
 	install -d $(DESTDIR)$(prefix)/bin
 	rm -f $(DESTDIR)$(prefix)/bin/fs-uae-launcher
 	ln -s ../share/fs-uae-launcher/fs-uae-launcher 	$(DESTDIR)$(prefix)/bin/fs-uae-launcher
